### PR TITLE
Add builders tab to show members

### PR DESCRIPTION
**Pull request by solo1g**
## Description

Adds a builders tab to show checked in members.

## Additional Information

- [x] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [x] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

## Related Issues

_Closes #3_

Your ENS/address: **solo1g.eth**

### CODE REVIEW

File: packages/nextjs/pages/builders.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+  const spinner = (
+    <div className="my-24 flex justify-center">
+      <Spinner width="50" height="50" />
+    </div>
+  );
+
+  const builderTile = (address: string) => {
+    return (
+      <a
```
Comment (carletex) [inline packages/nextjs/pages/builders.tsx:21]: For internal links, let's use NextJS `<Link>`

**Review by carletex — COMMENTED** [review]
Looking good @solo1g 

Left a couple of comments.

**Comment by solo1g** [comment]
Hi, changed as per comments.

**Review by carletex — APPROVED** [review]
Good stuff @solo1g

**Pull request merged**